### DISCUSSION TITLE
Where to add "!relationhips"?

### DISCUSSION THREAD

**Discussion by beffge** [structurizr/java#335] [Q&A] [answered]
Hi,

here's an example:
```
workspace {
    model {
        s1 = softwareSystem mySystem {
            c1 = container sourceContainer
            c2 = container destinationContainer
            c1 -> c2 "uses"
        }
        !relationships "relationship==sourceContainer->destinationContainer" {
            tags "added-tag"
        }
    }
}
```
No matter where I move the `!relationships` I can't get it to work. What am I missing?

Thanks for the help!

**Comment by beffge** [comment, marked as answer]
OK, found the problem 🙈 

https://github.com/structurizr/java/blob/9529188608e5ad08efc3a25a0fcea5d7933bcb96/changelog.md?plain=1#L14

@simonbrowndotje any plans on when this will be released? Since the docs ([already released](https://docs.structurizr.com/dsl/language#relationships)) but the code not 😅

**Reply by simonbrowndotje** [reply]
v3.0.0 is now released, along with a new Structurizr Lite version that includes it. CLI, cloud, and on-premises will follow.
